script.js

```diff
@@ -1,4 +1,194 @@
+  if (gameMode == 1) {
+    initialDraw();
+    console.log("Player Initial Hand: " + playerHand[0].rank + "," + playerHand[1].rank);
+    console.log("Computer Initial Hand: " + computerHand[0].rank + "," + computerHand[1].rank);
+    var playerHandSum = getHandSum(playerHand);
+    var computerHandSum = getHandSum(computerHand);
+    //if computer hand sum < 17, hit with additional card
+    while (computerHandSum < 17) {
+      getAdditionalCard(computerHand);
+      computerHandSum = getHandSum(computerHand);
+      console.log("Computer Final Hand: " + computerHandSum)
+    };
+    myOutputValue = "You were dealt " + playerHand[0].name + " of " + playerHand[0].suit + " and " + playerHand[1].name + " of " + playerHand[1].suit + ". Your current score is " + playerHandSum + ". Please type hit to draw additional card.";
+    console.log(gameMode);
+    gameMode = 2;
+    return myOutputValue;
+    // switch game mode to let player decide if want to hit or stand
+  };
+  //Player to decide whether to hit or stand
+  if (gameMode == 2) {
+    if (input == "hit") {
+      getAdditionalCard(playerHand);
+      console.log("Additional card: " + playerHand[playerHand.length - 1].rank);
+      playerHandSum = getHandSum(playerHand);
+      console.log("Player's Hand: " + playerHandSum);
+      
+      if (playerHandSum > blackJackLimit) {
+        myOutputValue = "You drew " + playerHand[playerHand.length - 1].name + " of " + playerHand[playerHand.length - 1].suit + ". Your current score is " + playerHandSum + ". You bust!"
+        return myOutputValue
+      }
+      else {
+      } myOutputValue = "You drew " + playerHand[playerHand.length - 1].name + " of " + playerHand[playerHand.length - 1].suit + ". Your current score is " + playerHandSum + ". Please type hit to draw another additional card. If not, please hit submit.";
+      return myOutputValue
+    } else if (input != "hit") {
+      gameMode = 3; 
+    }
+  }
+  // Comparing scores:
+  if (gameMode == 3) {
+    console.log(gameMode)
+    playerHandSum = getHandSum(playerHand)
+    computerHandSum = getHandSum(computerHand)
+    if (playerHandSum > blackJackLimit) {// if player > 21, player loses
+      myOutputValue = "You bust! Your total hand is " + playerHandSum + ". Try again.";
+    } else if (computerHandSum > blackJackLimit) {// else if computer > 21, computer loses
+      myOutputValue = "Dealer bust! Dealer's total hand is " + computerHandSum + ". You won!";
+    } else if (computerHandSum > playerHandSum) {
+      myOutputValue = "You lost! Dealer's total hand is " + computerHandSum + ". Your total hand is " + playerHandSum + ".";
+    } else if (playerHandSum > computerHandSum) {
+      myOutputValue = " You won! Dealer's total hand is " + computerHandSum + ". Your total hand is " + playerHandSum + ".";
+    } else if (playerHandSum == computerHandSum) {
+      myOutputValue = "It's a draw. Dealer's total hand is " + computerHandSum + ".";
+     
+    }
```

Comment:
When using else ifs do make sure to end with an else. 

It looks cleaner. Your else can also serve as a conditional to catch errors. E.g. if all of the conditions above are not met, then there must be a bug in the code